stratisd_cert.py

```diff
@@ -527,6 +527,21 @@ def test_filesystem_create(self):
             StratisDbus.fs_create(pool_path, fs_name), dbus.UInt16(0)
         )
 
+    def test_filesystem_create_specified_size(self):
+        """
+        Test creating a filesystem with a specified size.
+        """
+        pool_name = p_n()
+        pool_path = make_test_pool(pool_name, StratisCertify.DISKS[0:1])
+
+        fs_name = fs_n()
+        fs_size = "8796093022208"
+        kwargs = {"fs_size": fs_size}
+
+        self._unittest_command(
+            StratisDbus.fs_create(pool_path, fs_name, **kwargs), dbus.UInt16(0)
```

Comment:
It is not necessary to use the ```kwargs``` name trick here. It's useful in some situations but definitely overkill here since you can instead just use the expression
```StratisDbus.fs_create(pool_path, fs_name, fs_size="8796093022208")```

Reply:
OK; for some reason, what I was trying before wasn't working, and I was corralled into using kwargs.